src/pages/Task.tsx

```diff
@@ -62,6 +63,16 @@ export const Task: React.FC = () => {
     task: { displayName, version }
   } = data;
 
+  let BodyComp = <></>
```

Comment:
Does it make sense to refactor the tab contents here into a higher-order component? I can see this becoming somewhat messy as we add tabs, and a separate HOC might help with that